layouts/faq/list.html

```diff
@@ -1,16 +1,328 @@
+                         <h4 class="panel-title">
+                            <a href="#" class="ing">I wish to propose a collaboration and/or a project with the Orange team.</a>
+                      </h4>
+
+                    </div>
+                    <div id="question34" class="panel-collapse collapse" style="height: 0px;">
+                        <div class="panel-body">
+
+                            <p>Please submit a contact request and we will respond as soon as we can. </p>
+                        </div>
+                    </div>
+                </div>
+
+
+                <div class="panel panel-default ">
+                    <div class="panel-heading accordion-toggle collapsed question-toggle" data-toggle="collapse" data-parent="#faqAccordion2" data-target="#question32">
+                         <h4 class="panel-title">
+                            <a href="#" class="ing">I wish to develop a widget / an add-on for Orange.</a>
+                      </h4>
+
+                    </div>
+                    <div id="question34" class="panel-collapse collapse" style="height: 0px;">
+                        <div class="panel-body">
+
+                            <p>We are always happy to receive contributions! Development documentation for  <a href="http://orange-development.readthedocs.io/"> widgets</a> and <a href="https://github.com/biolab/orange3-example-addon">add-ons</a> is available on the web. Next, submit a pull request and our team will review it. If you have specific questions, use <a href="https://gitter.im/biolab/orange3">Gitter</a> for direct communication with the developers.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                 <div class="panel panel-default ">
+                    <div class="panel-heading accordion-toggle collapsed question-toggle" data-toggle="collapse" data-parent="#faqAccordion2" data-target="#question32">
+                         <h4 class="panel-title">
+                            <a href="#" class="ing">I wish to commission a custom module and/or an add-on for Orange.</a>
+                      </h4>
+
+                    </div>
+                    <div id="question34" class="panel-collapse collapse" style="height: 0px;">
+                        <div class="panel-body">
+
+                            <p>Please submit a contact request to discuss the idea and rates.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                
+                
+            </div>
+        </div>
+    
 </section>
+<script type="text/javascript">
+    
+function troubleshooting(){
+    $("#button2").removeClass("activefaq");
+    $("#button3").removeClass("activefaq");
+    $("#button4").removeClass("activefaq");
+    $("#education, #aboutus, #aboutorange").hide()
+    $("#troubleshooting").toggle();
```

Comment:
If the currently selected category is clicked again, it's deselected, leading to a state where no FAQ are displayed. Let it be so that a category is displayed at all times.

I think that changing `toggle()` and `toggleClass()` methods to `show()` and `addClass()` respectively should be sufficient.